apps/server/src/modules/files-storage/repo/filerecord.repo.integration.spec.ts

```diff
@@ -87,6 +87,7 @@ describe('FileRecordRepo', () => {
 			await em.persistAndFlush(fileRecord);
 			const origUpdatedAt = fileRecord.updatedAt;
 
+			// eslint-disable-next-line no-promise-executor-return
 			await new Promise((resolve) => setTimeout(resolve, 20));
```

Comment:
```suggestion
			await new Promise((resolve) => {
			    setTimeout(resolve, 20));
			}
```
Would this be the solution here instead of the eslint-disable?